webapp/commonComponents/profilePicture.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+import { useProfilePicture } from './hooks'
+
+export default ({ userUuid, forceUpdateKey = 0 }) => {
```

Comment:
would it be possible to have a component . and also assign defaultProps with some docs?
e.g. 
```
const ProfilePicture = props => ...

ProfilePicture.defaultProps = {.....

export default ProfilePicture
```

Reply:
ok